src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
+function CalculateRentalCost(days) {
```

Comment:
The function name `CalculateRentalCost` should be in camelCase, i.e., `calculateRentalCost`, to follow JavaScript naming conventions.